doc/api/convenience_functions.rst

```diff
@@ -0,0 +1,14 @@
+Convenience functions
+=====================
+
+.. py:currentmodule:: sounddevice
```

Comment:
You can skip the `py:` as it is the default.

Reply:
Done.